public_html/profiles/os2web/modules/os2web_accessibility/os2web_accessibility.module

```diff
@@ -165,13 +166,14 @@ function os2web_accessibility_theme() {
       'variables' => array(),
     ),
     'os2web_accessibility' => array(
-      'variables' => array(
+      'arguments' => array(
         'handicap' => array(),
         'type' => PRINT_HTML_FORMAT,
         'node' => NULL,
         'menu' => NULL,
         'right_blocks' => NULL
       ),
+      'path' => drupal_get_path('module', 'os2web_accessibility'),
```

Comment:
why is this line needed? Where the path is used?

Reply:
@stanbellcom I delete this line